### PR TITLE
refine roi_average_align_2d interface

### PR DESCRIPTION
**Pull request by knorth55**
related to #5304 
support new interface `roi_average_align_2d`
roi_average_align_2d

### CODE REVIEW

File: chainer/functions/pooling/roi_average_align_2d.py [hunk 1]
```diff
@@ -121,18 +121,20 @@ def _get_bilinear_interp_params(y, x, height, width):
...
-                raise TypeError(
-                    '{} must be positive integer: {}, {}'
-                    .format(arg, type(value), value)
-                )
+class ROIAverageAlign2D(function.Function):
+
+    """ROI average align over a set of 2d planes."""
+
+    def __init__(self, outsize, spatial_scale, sampling_ratio=0):
+        outh, outw = outsize
```
Comment (Hakuyume) [inline chainer/functions/pooling/roi_average_align_2d.py]: How about supporting an integer for `outsize`?

Comment (knorth55) [reply chainer/functions/pooling/roi_average_align_2d.py]: ok

Comment (knorth55) [reply chainer/functions/pooling/roi_average_align_2d.py]: Updated

**Comment by knorth55** [comment]
Update dosctring
roi_average_align_2d

**Comment by beam2d** [comment]
LGTM

**Pull request merged**